services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -110,6 +121,9 @@ class FlowHelper {
 
         checkRulesOnSwitches(flowEntry, RULES_DELETION_TIME, false)
 
+        switchPortsMap[flowEntry.left.sourceSwitch] -= flowEntry.left.sourcePort
```

Comment:
Given that this thing is managed by Spring and can survive between test cases, I'd rather add appropriate cleanup to the BaseSpecification.

Reply:
Agree.